solidity/test/token_stake/TestTopUps.js

```diff
@@ -843,5 +888,39 @@ describe('TokenStaking/TopUps', () => {
       delegationInfo = await tokenStaking.getDelegationInfo(operatorFour)
       expect(delegationInfo.amount).to.eq.BN(delegatedAmount.muln(2))
     })
+
+    it("can be done with less than the minimum stake", async () => {
+      await delegateStakeFromGrant(
+        tokenGrant,
+        tokenStaking.address,
+        grantee,
+        operatorFour,
+        beneficiary,
+        authorizer,
+        delegatedAmount,
+        grantId
+      )
+
+      await time.increase(initializationPeriod.addn(1))
+
+      let delegationInfo = await tokenStaking.getDelegationInfo(operatorFour)
+      expect(delegationInfo.amount).to.eq.BN(delegatedAmount)
+
+      const topUpAmount = 91
```

Comment:
It may be better to set this value relative to the minimum stake.
```suggestion
      const topUpAmount = (await tokenStaking.minimumStake()).subn(1)
```

Reply:
Please see https://github.com/keep-network/keep-core/pull/1974/commits/81619c70a48b3f53828a3f0894ddfbb78e00343e